textual-practices.csl

```diff
@@ -0,0 +1,372 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" default-locale="en-GB">
+  <info>
+    <title>Textual Practices</title>
+    <title-short>TextPrac</title-short>
+    <id>http://www.zotero.org/styles/textual-practices</id>
+    <link href="http://www.zotero.org/styles/textual-practices" rel="self"/>
+    <author>
+      <name>Jon Rubin</name>
+    </author>
+    <category citation-format="note"/>
+    <category field="humanities"/>
+    <issn>1121-2179</issn>
+    <eissn>1825-6562</eissn>
+    <summary>Textual Practices journal’s citation style: footnotes plus IBID citation style</summary>
+    <published>2014-01-24T00:00:00+00:00</published>
+    <updated>2018-05-22T00:59:23+00:00</updated>
```

Comment:
Could you changed the published date to when you made the style (something in Feb 2019) and remove the updated line?